cumulus_library/studies/core/builder_allergyintolerance.py

```diff
@@ -0,0 +1,66 @@
+import cumulus_library
+from cumulus_library.studies.core.core_templates import core_templates
+from cumulus_library.template_sql import sql_utils
+
+expected_table_cols = {
+    "allergyintolerance": {
+        "id": [],
+        "type": [],
+        "category": [],
+        "criticality": [],
+        "recordedDate": [],
+        "patient": sql_utils.REFERENCE,
+        "encounter": sql_utils.REFERENCE,
+        "reaction": ["severity"],
+    }
+}
+
+
+class CoreAllergyIntoleranceBuilder(cumulus_library.BaseTableBuilder):
+    display_text = "Creating AllergyIntolerance tables..."
+
+    def prepare_queries(self, *args, config: cumulus_library.StudyConfig, **kwargs):
+        code_sources = [
+            sql_utils.CodeableConceptConfig(
+                source_table="allergyintolerance",
+                column_hierarchy=[("clinicalStatus", dict)],
+                target_table="core__allergyintolerance_dn_clinical_status",
+                filter_priority=True,
+                code_systems=[
+                    # Restrict to just this required binding system
+                    "http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical",
```

Comment:
For this and other code filters: Can we check Cerner data to see if it is providing these codes back? Or what the epic sandbox is doing?

Reply:
`clinicalStatus` and `verificationStatus` are a few of the rare Required bindings in the base spec. That's why I'm being so restrictive/simplistic here. These CodeableConcept fields are basically glorified (read: complicated) Code fields. Every other resource (except Condition) uses simple Code fields for status values, instead of a required-binding-CodeableConcept. :shrug: Either one of those weird "two people working on different parts of spec" things, or they anticipated some local expansions of the status values.

So support is good in EHRs. I checked against sample exports from Epic & Cerner - they both provide it.

Reply:
ok, good deal